docs/en/observability/apm.asciidoc

```diff
@@ -1,6 +1,10 @@
 [[apm]]
 = Application performance monitoring (APM)
 
+++++
+<titleabbrev>Application performance (APM)</titleabbrev>
```

Comment:
It seems strange to use the acronym "APM" when the full text doesn't include the "M" word.

Reply:
@colleenmcginnis Do you think it's better to just remove APM or spell it all out? I'm trying to keep the nav titles relatively short, if possible.

Reply:
I'm inclined to say we should spell it out because I think "Application performance monitoring" as a phrase is more universally recognizable. I'm curious what @bmorelli25 thinks.

Reply:
I'm gonna remove the short title for now and see how other reviewers respond.